doc/relevancy.md

```diff
@@ -0,0 +1,459 @@
+# Relevancy
+
+This document explains how relevancy ordering works when performing a
+search.
+
+
+## Contents
+
+1. [What is relevancy?](#what-is-relevancy)
+2. [What impacts relevancy?](#what-impacts-relevancy)
+   1. [Boosting](#boosting)
+   2. [Best and worst bets](#best-and-worst-bets)
+   3. [Stopwords](#stopwords)
+   4. [Synonyms](#synonyms)
+   5. [Categorisation of fields](#categorisation-of-fields)
+   6. [Analyzers](#analyzers)
+   7. [Excluded formats](#excluded-formats)
+3. [Possible problems with queries and relevance](#possible-problems-with-queries-and-relevance)
+
+
+## What is relevancy?
+
+A list of documents returned by Search API will include an `es_score`
+on every document.
+
+```ruby
+# Response for a search for 'Harry Potter'
+[
+  { title: "Harry Potter", es_score: 1 },
+  { title: "Harry Kane", es_score: 0.5 },
+  { title: "Ron Weasley", es_score: 0.05 }
+]
+```
+
+The `es_score` value is used for ranking results and represents how
+relevant we think a result is to your query.
+
+### Debugging es_score
+
+If you want to understand why a result has a given `es_score`, you can
+use the Elasticsearch [Explain API][explain].  This is exposed by the
+Search API.
+
+You can see the reasons behind an `es_score` by including the
+`debug=explain` query parameter in your query.  This will add an
+`_explanation` field to every result, similar to a SQL-like `EXPLAIN`.
+
+For example, see the explanation produced by [searching for "harry
+potter"][explain-example].  This shows an example of stemming, where
```

Comment:
Where is this explanation? I had a look through the URL linked here and couldn't see it, maybe worth clarifying?

Reply:
There is an `_explanation` field which displays the source of each result's `es_score`. It's generated by the [Explain API](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-explain.html).

Reply:
Ah, okay, I see it now.